ga4gh/backends.py

```diff
@@ -186,19 +186,19 @@ def searchVariants(self, request):
         else:
             readCols = self._table.columns()[:self._firstSamplePosition]
             callSetIds = request.callSetIds
-            for csid in request.callSetIds:
-                cols = [c for name, c in self._sampleCols[csid]]
+            for callSetId in request.callSetIds:
+                cols = [col for name, col in self._sampleCols[callSetId]]
                 readCols.extend(cols)
         # Now we get the row positions for the sample columns
```

Comment:
@Naburimannu, wouldn't it be easier to just select it by index like this:

```
cols = list(self._sampleCols[callSetId][0])[1:]
```

The code looks really nice, but it would be great to add more documentation regarding the process that happens behind the scenes.  Maybe something like this might be nice:

```
# The selection is performed to return a list(tuple) for the column of interest
#   for the specificed callSetId from self._sampleCols, as such:
#  
#   [(u'GT', <wormtable.Column object at 0x2aaab7d70090>)]
#
# Then the required information is a list containing only the wormtable column, as follows:
#
#   [<wormtable.Column object at 0x2aaab7d70090>]
```

Just a thought,
~p